camel/model_backend.py

```diff
@@ -81,9 +108,24 @@ class ModelFactory:
     """
 
     @staticmethod
-    def create(model_type: ModelType, model_config_dict: Dict) -> ModelBackend:
+    def create(model_type: Optional[ModelType],
+               model_config_dict: Dict) -> ModelBackend:
+        r"""Creates an instance of `ModelBackend` of the specified type.
+
+        Args:
+            model_type (ModelType): Model for which a backend is created.
+            model_config_dict (Dict): a dictionary that will be fed into
+                the backend constructor.
+
+        Raises:
+            ValueError: If there is not backend for the model.
+
+        Returns:
+            ModelBackend: The initialized backend.
+        """
         default_model_type = ModelType.GPT_3_5_TURBO
```

Comment:
```suggestion
        model_type = model_type or ModelType.GPT_3_5_TURBO
```

Reply:
This sort of syntactic sugar is difficult to understand. Also it may lead to hidden errors like in this example:
https://stackoverflow.com/questions/3914667/false-or-none-vs-none-or-false
I'd rather not to.